packages/react-router/index.ts

```diff
@@ -468,45 +337,33 @@ export { ServerMode as UNSAFE_ServerMode } from "./lib/server-runtime/mode";
 // extreme caution and only if you understand the consequences. Godspeed.
 ///////////////////////////////////////////////////////////////////////////////
 
+/** @internal */
+export { ErrorResponseImpl as UNSAFE_ErrorResponseImpl } from "./lib/router/utils";
+
+/** @internal */
+export {
+  decodeViaTurboStream as UNSAFE_decodeViaTurboStream,
+  SingleFetchRedirectSymbol as UNSAFE_SingleFetchRedirectSymbol,
+} from "./lib/dom/ssr/single-fetch";
+
 /** @internal */
 export {
   DataRouterContext as UNSAFE_DataRouterContext,
   DataRouterStateContext as UNSAFE_DataRouterStateContext,
   LocationContext as UNSAFE_LocationContext,
   NavigationContext as UNSAFE_NavigationContext,
   RouteContext as UNSAFE_RouteContext,
-  mapRouteProperties as UNSAFE_mapRouteProperties,
-  useRouteId as UNSAFE_useRouteId,
-  useRoutesImpl as UNSAFE_useRoutesImpl,
-  UNSAFE_ErrorResponseImpl,
-};
-
-/** @internal */
-export { FrameworkContext as UNSAFE_FrameworkContext } from "./lib/dom/ssr/components";
-
-/** @internal */
-export type { RouteModules as UNSAFE_RouteModules } from "./lib/dom/ssr/routeModules";
```

Comment:
I removed most of the `UNSAFE_` exports at the bottom here, leaving only the contexts and then the few things we need exported for integration tests